### ISSUE TITLE
uncaughtException - TypeError: Cannot read property 'status' of undefined

### ISSUE THREAD

**Issue by HaroldStruwig** [happner/happn-3#167]
**TR4127: https://github.com/orgs/FieldServer/projects/218**

```
2018-11-07 17:37:31.570 [FATAL] - 3ms	serverMesh (Mesh) uncaughtException (or set NO_EXIT_ON_UNCAUGHT) { TypeError: Cannot read property 'status' of undefined
   at HappnClient.handle_publication (node_modules/happn-3/lib/client.js:1408:23)
   at done (primus.js:2673:48)
   at transform (primus.js:2644:30)
   at Primus.transforms (primus.js:2665:4)
   at decoding (primus.js:2428:14)
   at Primus.decoder (primus.js:3254:3)
   at Primus.message (primus.js:2417:12)
   at next (primus.js:143:17)
   at Primus.parse (primus.js:3204:7)
   at WebSocket.emitter (primus.js:146:12)
   at WebSocket.onMessage (node_modules/ws/lib/WebSocket.js:418:14)
   at Receiver.ontext (node_modules/ws/lib/WebSocket.js:816:10)
   at node_modules/ws/lib/Receiver.js:477:18
   at Receiver.applyExtensions (node_modules/ws/lib/Receiver.js:364:5)
   at node_modules/ws/lib/Receiver.js:466:14
   at Receiver.flush (node_modules/ws/lib/Receiver.js:340:3)
   at Receiver.finish (node_modules/ws/lib/Receiver.js:482:12)
   at Receiver.expectHandler (node_modules/ws/lib/Receiver.js:457:31)
   at Receiver.add (node_modules/ws/lib/Receiver.js:95:24)
   at Socket.realHandler (node_modules/ws/lib/WebSocket.js:800:20)
   at addChunk (_stream_readable.js:263:12)
   at readableAddChunk (_stream_readable.js:250:11)
   at Socket.Readable.push (_stream_readable.js:208:10)
   at TCP.onread (net.js:601:20) uncaught: true }
```

**Comment by HaroldStruwig** [comment]
only happens when `encryptPayloads` is enabled

**Comment by HaroldStruwig** [comment]
reproducable on repo https://github.com/FieldServer/happner-endpoint - branch 'issue-56' - the two skipped tests both have this issue

**Comment by JohanvdWest** [comment]
Closed by #165
